Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -695,6 +698,8 @@ private void FindOrCreateUiRaycastCamera()
             GameObject cameraObject = null;
 
             var existingUiRaycastCameraObject = GameObject.Find("UIRaycastCamera");
+            bool uiRaycastCameraObjectExists = existingUiRaycastCameraObject != null;
+            Debug.Assert(uiRaycastCameraObjectExists == MixedRealityToolkit.Instance.IsProfileSwitching);
```

Comment:
Is this assertion always true? For example, if I decide to put a specific UIRaycastCamera in my scene before building, so MRTK doesn't have to create one at runtime, it seems like this assertion will fail.

Reply:
I did not know that there would be a reason for someone to do that, so will remove the assertion.